src/browserview/BrowserComponent.java

```diff
@@ -526,4 +526,12 @@ public void showContributors() {
 			}
 		});
 	}
+
+	public boolean isCurrentUrlIssue() {
+		if (driver.getCurrentUrl().matches("https://github.com/(.+)/(.+)/(issues|pull)/([0-9]+)")) {
+			return true;
+		}
+		return false;
+	}
+
```

Comment:
Hmm. good catch on the url. However i still need to check for url via browsercomponent as i need to know what the url is. Check out the latest commit :)